Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM --platform=$BUILDPLATFORM golang:1.21 AS builder
+ARG golang_image=golang:1.21
 
+FROM --platform=$BUILDPLATFORM $golang_image AS builder
 WORKDIR $GOPATH/src/github.com/aws/amazon-eks-pod-identity-webhook
 COPY . ./
+RUN go version
```

Comment:
Is this for debugging?

Reply:
> Is this for debugging?

No, this is intentional, we want to print out the go version explicitly